quarkus-opentelemetry-exporter-sentry/runtime/src/main/java/io/quarkiverse/opentelemetry/exporter/sentry/beans/SentrySpanProcessorProducer.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkiverse.opentelemetry.exporter.sentry.beans;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.enterprise.inject.Produces;
+
+import io.sentry.opentelemetry.SentrySpanProcessor;
+
+public final class SentrySpanProcessorProducer {
+    @SuppressWarnings("unused")
+    @ApplicationScoped
+    @Produces
+    public SentrySpanProcessor produceSentrySpanProcessor() {
+        return new SentrySpanProcessor();
```

Comment:
Please rename, `SentrySpanProcessor` looks like a processor from the deployment module.

Reply:
got it!